ehr/resources/views/ehrAdmin.html

```diff
@@ -1,96 +1,44 @@
 <script type="text/javascript">
```

Comment:
Do any current EHR modules use this implementation? If so, they probably won't want to start showing admin links to all users. But if not, then it's fine.

Reply:
All other EHR clients override this view.  The view.xml also sets permission to admin.